.github/workflows/benchmarking.yml

```diff
@@ -113,3 +116,91 @@ jobs:
         slack-message: "Periodic perf testing: ${{ job.status }}\n${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+  bench-captest:
+    env:
+      TEST_PG_BENCH_DURATIONS_MATRIX: "60m"
+      TEST_PG_BENCH_SCALES_MATRIX: "10gb"
+      REMOTE_ENV: "1"
+      POSTGRES_DISTRIB_DIR: /usr
+
+    strategy:
+      fail-fast: false
+      matrix:
+        connstr: [ BENCHMARK_CAPTEST_CONNSTR, BENCHMARK_RDS_CONNSTR ]
+
+    runs-on: dev
+    timeout-minutes: 360 # 6h
+
+    container:
+      image: python:3.9-buster
+
+    steps:
+    - name: Checkout Neon repo
+      uses: actions/checkout@v3
+
+    - name: Install Deps
+      run: |
+        # Newer git required for submiting results
+        echo "deb http://deb.debian.org/debian buster-backports main" > /etc/apt/sources.list.d/buster-backports.list
+
+        echo "deb http://apt.postgresql.org/pub/repos/apt buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+        wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+        apt -y update
+        apt install -y postgresql-14 -t buster-backports git
+
+        psql --version
+        pgbench --version
+        git --version
+
+    - name: Install poetry & python deps
+      run: |
+        python3 -m pip install --upgrade poetry
+        ./scripts/pysync
+
+    - name: Benchmark init
+      env:
+        PLATFORM: ${{ (matrix.connstr == 'BENCHMARK_RDS_CONNSTR' && 'rds-aurora') || 'neon-captest' }}
+        BENCHMARK_CONNSTR: ${{ secrets[matrix.connstr] }}
+      run: |
+        mkdir -p perf-report-captest
+
+        psql $BENCHMARK_CONNSTR -c "SELECT 1;"
+        ./scripts/pytest test_runner/performance/test_perf_pgbench.py -v -m "remote_cluster" -k "init" --skip-interfering-proc-check --out-dir perf-report-captest --timeout 21600
```

Comment:
Because the name also depends on other parameters, it will require to be changed if we change the duration or DB size. And I will definitely forget about it. So I think I'll keep it as it is.

Reply:
Ah yes that makes sense. It can still backfire, but it's better.

Let's come back to this after the readme is updated. I'd ask "why not run **all** remote tests, in one step, just like we do against staging", but I'll get my answer from the readme :)

Reply:
> why not run all remote tests, in one step, just like we do against staging

Long story short: Because we want to run them separately 😄 

This PR is a part of https://github.com/neondatabase/neon/issues/2016. 
Initially, the PR was about adding like 300GB database, but during the process of setting it up, I've undercovered a couple of issues[1], [2] that make it hardly possible to do with pgbench initialize. 
I've split the test into steps to make it possible to set up the database once and then run only `simple-update`/`select-only` flows. Also, such separation makes it more convenient to see what's going on when you open a GitHub workflow run (is it still initializing or doing some real tests).

And this PR has evolved into something like, "let's start collecting initialize results and compare them with RDS, then we will be able to set up a big database as it was intended originally"

- [1] https://github.com/neondatabase/cloud/issues/1872
- [2] https://github.com/neondatabase/neon/issues/2221